src/tests/ftest/server/metadata.py

```diff
@@ -412,18 +402,15 @@ def test_metadata_server_restart(self):
                     len(errors), 0,
                     "Error stopping agents:\n  {}".format("\n  ".join(errors)))
 
-                # Stop the servers
-                errors = self.stop_servers()
+                # Restart the servers w/o formatting the storage
+                errors = self.restart_servers()
```

Comment:
Any reason why we are changing the test steps here? stop_servers and start_servers are going to be different from restart_servers behaviour... Is it not? Unless restart_servers does the same thing as stop and start servers.

Reply:
The problem with `self.start_server_managers()` is that it formats the storage before starting the servers so you lose the data written by ior (in the first loop) and then the ior read (in the second loop) has no data to read.  This change resolves this by excluding the storage format.